tv-shows-app/src/components/shared/ShowListContainer/ShowListContainer.tsx

```diff
@@ -12,16 +14,16 @@ interface IShowListContainerProps {
 export default function ShowListContainer({
   topRated,
 }: IShowListContainerProps) {
-  const {
-    data: showListResponse,
-    error,
-    isLoading,
-  } = useSWR(
-    topRated ? "/api/shows/top-rated" : "/api/shows",
-    topRated ? getTopRatedShows : getShows
+  const { trigger, data, error, isMutating } = useSWRMutation(
```

Comment:
I see you switching to useSwrMutation everywhere while we should really use it only for requests that mutate data on the server. This is a regular fetch.